docs/src/components/MdxPage/MdxPageTabs/MdxPageTabs.tsx

```diff
@@ -65,6 +66,17 @@ export function MdxPageTabs({ body, frontmatter, headings }: MdxPage) {
               style={{ maxWidth: 1080, margin: 'auto', marginTop: 24 }}
               className={classes.tabContent}
             >
+              <Input
+                variant="default"
+                placeholder="Search Props"
+                style={{
+                  marginBottom: 20,
+                }}
+                value={searchValue}
+                onChange={(event: { currentTarget: { value: React.SetStateAction<string> } }) =>
```

Comment:
When Input is replaced with TextInput onChange event type definition will not be required, remove it